stacker/providers/aws/interactive.py

```diff
@@ -127,17 +129,23 @@ def _wait_till_change_set_complete(self, change_set_id):
 
     def update_stack(self, fqn, template_url, parameters, tags, **kwargs):
         logger.debug("Attempting to create change set for stack: %s.", fqn)
-        response = retry_on_throttling(
-            self.cloudformation.create_change_set,
-            kwargs={
-                'StackName': fqn,
-                'TemplateURL': template_url,
-                'Parameters': parameters,
-                'Tags': tags,
-                'Capabilities': ["CAPABILITY_NAMED_IAM"],
-                'ChangeSetName': get_change_set_name(),
-            },
-        )
+        try:
+            response = retry_on_throttling(
+                self.cloudformation.create_change_set,
+                kwargs={
+                    'StackName': fqn,
+                    'TemplateURL': template_url,
+                    'Parameters': parameters,
+                    'Tags': tags,
+                    'Capabilities': ["CAPABILITY_NAMED_IAM"],
+                    'ChangeSetName': get_change_set_name(),
+                    'NotificationARNs': [self.listener.TopicArn]
+                },
+            )
+        except botocore.exceptions.ClientError as e:
+            if "does not exist" in e.message:
+                raise exceptions.StackDoesNotExist(fqn)
```

Comment:
Need to just `raise` after this, so any other types of ClientError's don't get missed.